os/kernel/sched/sched_addreadytorun.c

```diff
@@ -125,11 +125,11 @@ bool sched_addreadytorun(FAR struct tcb_s *btcb)
 
 #ifdef CONFIG_SW_STACK_OVERFLOW_DETECTION
 	if (*(uint32_t *)(rtcb->stack_base_ptr) != STACK_COLOR) {
-		dbg_noarg("###############    STACK OVERFLOW at pid %d ", rtcb->pid);
+		lldbg_noarg("\n###############    STACK OVERFLOW at pid %d ", rtcb->pid);
 #if CONFIG_TASK_NAME_SIZE > 0
-		dbg_noarg("(%s) ", rtcb->name);
+		lldbg_noarg("(%s) ", rtcb->name);
 #endif
-		dbg_noarg("###################\n");
+		lldbg_noarg("###################\n");
```

Comment:
It could cause log mix. 
Because we will call the PANIC after this message, how about adding enter_critical_section?